src/crawler.ts

```diff
@@ -2739,13 +2739,11 @@ self.__bx_behaviors.selectMainBehavior();
 
     this.lastSaveTime = now.getTime();
 
-    const ts = now.toISOString().slice(0, 19).replace(/[T:-]/g, "");
-
     const crawlDir = path.join(this.collDir, "crawls");
 
     await fsp.mkdir(crawlDir, { recursive: true });
 
-    const filenameOnly = `crawl-${ts}-${this.crawlId}.yaml`;
+    const filenameOnly = `crawl-${this.crawlId}.yaml`;
```

Comment:
This ends up writing files named like `crawl-7fd3b6baf75f-crawl-20250401141145058.yaml` by default. Maybe we drop the initial `crawl-` prefix?

```suggestion
    const filenameOnly = `${this.crawlId}.yaml`;
```

Reply:
Changed this to be `$ts-$id.yaml` to match previous pattern.

Reply:
We actually do want a unique ts here as well, as its a new save state

Reply:
ah true!